src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_PlasmaForge.java

```diff
@@ -1106,6 +1120,32 @@ protected ResourceLocation getActivitySoundLoop() {
         return SoundResource.GT_MACHINES_PLASMAFORGE_LOOP.resourceLocation;
     }
 
+    @Override
+    public byte getUpdateData() {
+        byte data = 0;
+        if (discount == maximum_discount) {
+            data += DISCOUNT_BITMAP;
+        }
+        if (convergence) {
+            data += CONVERGENCE_BITMAP;
+        }
+        return data;
+    }
+
+    @Override
+    public void receiveClientEvent(byte aEventID, byte aValue) {
+        if (aEventID == 1) {
+            if ((aValue & CONVERGENCE_BITMAP) == CONVERGENCE_BITMAP) {
+                convergence = true;
+            } else {
+                convergence = false;
+            }
```

Comment:
Please, use clear event identifier - no need for magic numbers.
```suggestion
        if (aEventID == GregTechTileClientEvents.CHANGE_CUSTOM_DATA) {
            convergence = (aValue & CONVERGENCE_BITMAP) == CONVERGENCE_BITMAP;
```